setup.py

```diff
@@ -1,45 +1,41 @@
-"""OpenSCM
+"""
+OpenSCM
+-------
+
+A unifying interface for Simple Climate Models.
+
 """
 
 import versioneer
 
 from setuptools import setup, find_packages
 from setuptools.command.test import test as TestCommand
 
-PACKAGE_NAME = "openscm"
-AUTHOR = "Robert Gieseke"
-EMAIL = "robert.gieseke@pik-potsdam.de"
-URL = "https://github.com/openclimatedata/openscm"
-
-DESCRIPTION = "A unifying interface for Simple Climate Models"
-README = "README.rst"
 
 REQUIREMENTS = [
     "numpy",
     "pint",
     "pandas",
-    # can be moved into notebooks dependencies once Jared's new backend is in place
+    # TODO can be moved into notebooks dependencies once Jared's new backend is in place
     "pyam-iamc @ git+https://github.com/IAMconsortium/pyam.git@master",
 ]
 REQUIREMENTS_NOTEBOOKS = ["notebook", "seaborn"]
 REQUIREMENTS_TESTS = ["codecov", "nbval", "pytest", "pytest-cov"]
 REQUIREMENTS_DOCS = ["sphinx>=1.4", "sphinx_rtd_theme", "sphinx-autodoc-typehints"]
 REQUIREMENTS_DEPLOY = ["setuptools>=38.6.0", "twine>=1.11.0", "wheel>=0.31.0"]
 
-requirements_dev = [
-    *["flake8", "black"],
-    *REQUIREMENTS_NOTEBOOKS,
-    *REQUIREMENTS_TESTS,
-    *REQUIREMENTS_DOCS,
-    *REQUIREMENTS_DEPLOY,
-]
-
-requirements_extras = {
+REQUIREMENTS_EXTRAS = {
     "notebooks": REQUIREMENTS_NOTEBOOKS,
     "docs": REQUIREMENTS_DOCS,
     "tests": REQUIREMENTS_TESTS,
     "deploy": REQUIREMENTS_DEPLOY,
-    "dev": requirements_dev,
+    "dev": [
```

Comment:
Why not use the `requirements_dev` variable?

Reply:
matter of taste like the issue above, i suppose. waiting for @rgieseke's decision ;)